src/app/users/users.service.ts

```diff
@@ -25,4 +25,40 @@ export class UsersService {
     return this.http.get('/users');
   }
 
+  /**
+   * @param {any} user User to be created.
+   * @returns {Observable<any>}
+   */
+  createUser(user: any): Observable<any> {
+    return this.http.post('/users', user);
+  }
+
+  /**
+   * @returns {Observable<any>} Offices data
+   */
+  getOffices(): Observable<any> {
+    return this.http.get('/offices');
+  }
+
+  /**
+   * @param {any} officeId ID of office to retrieve staff from.
+   * @returns {Observable<any>} Staff.
+   */
+  getStaff(officeId: any = ''): Observable<any> {
```

Comment:
```suggestion
  getStaff(officeId: any = ''): Observable<any> {
```
Why do we need a default value of blank ('') here?

Reply:
It is so that all staff can be queried if required, I was planning ahead incase this function wanted to be reused in another component so I made sure that the function works even without inputting the officeId, which will return all staff regardless of office.

Reply:
I have an if statement which only sets the officeId to the request IF officeId is not blank. If the no parameters are passed into the function, officeId will be blank and it will not filter the query based on office.

Reply:
You can remove that default value and if statement for now, we can update later if required.